src/AWS.Deploy.Common/Recipes/Validation/IRecipeValidator.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AWS.Deploy.Common.Recipes.Validation
+{
+    public interface IRecipeValidator
```

Comment:
could you be consistent with the rest of the public interfaces and add some documentation to this?

Reply:
Fixed